core/src/game_state.rs

```diff
@@ -1061,7 +1087,12 @@ impl PlayPhase {
         }
 
         let (non_landlord_level_bump, landlord_level_bump, landlord_won) =
-            Self::compute_level_deltas(self.num_decks, non_landlords_points);
+            Self::compute_level_deltas(
+                self.num_decks,
+                non_landlords_points,
+                self.propagated.bonus_level_policy,
+                self.landlords_team.len() < self.propagated.players.len() / 2,
```

Comment:
there is an explicit (overridable) setting for `num_friends`, so we should use that instead of recomputing it here -- the rule is a bonus for "smaller than expected" team, not "smaller than default"

Reply:
good catch. Will fix. Thanks.